tests/conftest.py

```diff
@@ -342,3 +344,40 @@ def pytest_report_header():
         "PennyLane-Lightning Test Suite",
         f"::: Device: {device_name:<17} :::",
     ]
+
+
+# Extract _default_rng_seed from `rng_salt` in pytest.ini
+# which are used for generating random vectors/matrices in functions below
+config = configparser.ConfigParser()
+pytest_ini_path = os.path.join(os.path.dirname(__file__), "pytest.ini")
+read_files = config.read(pytest_ini_path)
+config.read("pytest.ini")
+rng_salt = config["pytest"]["rng_salt"]
+_default_rng_seed = int(hashlib.sha256(rng_salt.encode()).hexdigest(), 16)
+
+
+def get_random_matrix(n, seed=None):
```

Comment:
It would be good to add a small docstring that explains what `n` is (notably that it's the dimension and not the number of qubits)

Reply:
(same comment for next 3 functions)